workflows/bacterial_genomics/bacterial_genome_annotation/bacterial_genome_annotation-tests.yml

```diff
@@ -8,51 +8,51 @@
     Select a AMRFinderPlus database: amrfinderplus_V3.12_2024-05-02.2
   outputs:
     integronfinder2_logfile_text:
-      assert:
+      asserts:
         has_text:
           text: "Writing out results for replicon"
     integronfinder2_summary:
-      assert:
+      asserts:
         has_n_columns:
           n: 6
     integronfinder2_results_tabular:
-      assert:
+      asserts:
         has_n_columns:
           n: 14
     bakta_hypothetical_tabular:
-      assert:
+      asserts:
         has_n_columns:
           n: 9
     bakta_annotation_json:
-      assert:
+      asserts:
         has_text:
           text: "aa_hexdigest"
     bakta_annotation_tabular:
-      assert:
+      asserts:
         has_n_columns:
           n: 9
     isescan_results_tabular:
-      assert:
+      asserts:
         has_n_columns:
           n: 24
     isescan_summary_tabular:
-      assert:
+      asserts:
         has_text:
           text: "nIS"
     isescan_logfile_text:
-      assert:
+      asserts:
         has_text:
           text: "Both complete and partial IS elements are reported."
     plasmidfinder_result_json:
-      assert:
+      asserts:
         has_text:
           text: "positions_in_contig"
     plasmidfinder_results_tabular:
-      assert:
+      asserts:
         has_n_columns:
           n: 8
     tooldistillator_summarize:
-      assert:
+      asserts:
```

Comment:
below there are multiple 'has_text' and only the last one will be tested, right?